Source/Core/DolphinWX/RmObjAddEdit.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+#pragma once
+
+class CRmObjAddEdit : public wxDialog
```

Comment:
This class  (and file, really, as well the cpp file) can have a more descriptive name. `CRmObj` is a very vague starting title to anyone who looks at this for the first time.